src/base/system_parameter.c

```diff
@@ -8089,6 +8382,35 @@ sysprm_generate_new_value (SYSPRM_PARAM * prm, const char *value, bool check, bo
     }
 #endif /* SERVER_MODE */
 
+  if (strcasecmp (value, PRM_VALUE_DEFAULT) == 0)
+    {
+      set_default = true;
```

Comment:
Yes. It looks like the parameter was used only in one place, and it checked the same thing (the value to be "DEFAULT")